src/main/java/uk/gov/ons/census/casesvc/service/CCSPropertyListedService.java

```diff
@@ -78,22 +54,10 @@ public void processCCSPropertyListed(
         messageTimestamp);
   }
 
-  private void handleUacQidLinksForCase(List<UacDTO> qids, Case caze) {
-    for (UacDTO qid : qids) {
-      addUacLinkForQidAndCase(qid.getQuestionnaireId(), caze);
-    }
-  }
-
-  private void sendActiveCCSCaseToField(Case caze) {
-    if (caze.getRefusalReceived() == null && !caze.isAddressInvalid()) {
+  private void sendCCSCaseToFieldIfInterviewRequired(Case caze, boolean interviewRequired) {
```

Comment:
having a function just to look at a well-named boolean seems pointless

Reply:
yeah fair enough, just sort of left it there